x/blackbox/dryrun_mixin_docs_test.py

```diff
@@ -0,0 +1,612 @@
+"""
```

Comment:
New test, based on old test in docs. Helps ensure that refactoring won't break origin `DryrunTestCaseMixin`

Reply:
To confirm, this file does not have anything to do with the new blackbox testing?

Reply:
Confirmed: I added it to ensure that refactoring `DryRunTestMixin` did not break existing functionality.